src/cppmangle.d

```diff
@@ -893,6 +893,13 @@ static if (TARGET_LINUX || TARGET_OSX || TARGET_FREEBSD || TARGET_OPENBSD || TAR
                 store(null);
             store(t);
         }
+
+        final const(char)* mangle_typeinfo(Dsymbol s)
+        {
+            buf.writestring("_ZTI");
```

Comment:
It is very likely that I have fixed this by using ``do_catch`` rather than pointer comparison.